### DISCUSSION TITLE
Possible broken bill?

### DISCUSSION THREAD

**Discussion by theRolento** [firefly-iii/firefly-iii#9046] [Firefly III]
![immagine](https://github.com/user-attachments/assets/b3e90fd4-9d64-4bd8-8b34-262d27f6d8cd)
The next bill should be on August 12th but the **Next expected match** field is blank and the **Repeats** is not updated.
Only this bill happens to be "possibly broken" apparently, but it's the only monthly bill i have at the moment (the other are yearly).

Normal behavior or there's a way to fix it?

**Comment by JC5** [comment]
The bill ends on July 12th, so future dates aren't calculated.

**Reply by theRolento** [reply]
> The bill ends on July 12th, so future dates aren't calculated.

I've modified the bill and removed the "end date" but still not calculated.
It's the only one that gives me problem

![immagine](https://github.com/user-attachments/assets/a9d80692-8f41-4bf0-ba93-1668d5eb81a4)

**Comment by theRolento** [comment]
I've delete the bill and recreated from scratch, blank "extend/end" dates this time (i edited the others, understood that they're optional).
The expected field still blank.
![immagine](https://github.com/user-attachments/assets/fa715bd9-df9a-4033-9c9f-e47f862145e5)
But if i open the bill the next expected match is there this time:
![immagine](https://github.com/user-attachments/assets/92a840d4-62be-4e92-ae79-8c016b5ce414)

**Reply by JC5** [reply]
Ah, but the bill was paid already in this period. It should show a date again after August 12th.
